quality/oracleapi/oracle.go

```diff
@@ -20,7 +20,7 @@ func New(url string) *API {
 	return &API{
 		url: url,
 		client: &http.Client{
-			Timeout: 5 * time.Second,
+			Timeout: 4 * time.Second,
```

Comment:
Changing from one magic number to another? 🤷🏻

Reply:
Initial TCP SYN retransmission is exactly 1 second (hardcoded in kernel). We may allow 2-3 attempts and about 1 second of interaction after it. It's quite widespread magic for queries expected to be fast.